wolfcrypt/src/sha256.c

```diff
@@ -1675,6 +1683,13 @@ static int InitSha256(wc_Sha256* sha256)
     #if defined(WOLFSSL_KCAPI_HASH)
         KcapiHashFree(&sha224->kcapi);
     #endif
+    #if defined(WOLFSSL_RENESAS_RX64_HASH)
+    wolfssl_RX64_HW_Hash* hw_sha = (wolfssl_RX64_HW_Hash*)sha224;
```

Comment:
Please fix mixed variable declaration. The variable needs to be declared at top of function or in a brace `{}` section.